src/validation/validationHealthcareProfessional.ts

```diff
@@ -217,7 +217,7 @@ export function validateDegrees(
         return
     }
 
-    if (degrees && degrees.length > 64) {
+    if (degrees && degrees.length > 16) {
```

Comment:
The 16 logic comes from how sensei Philip has implemented  when I started working on it, so I kept that for all validations. (Try to googled it but couldn't find mush info, probably because I don't use the right keywords)